src/hal/src/lib.rs

```diff
@@ -90,6 +90,10 @@ pub type InstanceCount = u32;
 pub type DrawCount = u32;
 /// Number of work groups.
 pub type WorkGroupCount = [u32; 3];
+///
+pub type TaskCount = u32;
+///
+pub type TaskOffset = u32;
```

Comment:
We have a separate `VertexOffset` because it has a different type, while the index and instance are just using "count" for both the count and the offset. Would be good to make this somewhat consistent. Perhaps, we could use `TaskCount` for both?

Reply:
Done.